resources/sample_vaults/Tasks-Demo/Styling/Styling of Queries.md

```diff
@@ -0,0 +1,40 @@
+# Styling of Queries
+
+To test styling of queries, follow this:
+
+- [ ] **1. Open the Obsidian settings of the Demo vault and under Appearance | CSS Snippets, turn on `tasks-plugin-query-styling`.**
```

Comment:
Does it need to say to switch to Reading mode or Live Preview?
Or should it be tested in both?

Reply:
> Does it need to say to switch to Reading mode or Live Preview?
> Or should it be tested in both?

Done in 30f31e77fbda3c1da2abff0202aa55f97b93145b